src/helpers/helpers.options.js

```diff
@@ -61,17 +61,18 @@ export function calculateTextAlignment(size, options) {
 }
 
 /**
- * @param {LabelPositionObject|string} value
- * @returns {LabelPositionObject}
+ * @param {{x: number|string, y: number|string}|string|number} value
+ * @param {string|number} defaultValue
+ * @returns {{x: number|string, y: number|string}}
  */
-export function toPosition(value) {
+export function toPointOption(value, defaultValue) {
```

Comment:
to me the old name is better. could also have a default value for the defaultValue :)
```suggestion
export function toPosition(value, defaultValue = 'center') {
```